doc/articles/uno-builds-troubleshooting.md

```diff
@@ -64,3 +64,7 @@ To generate a binlog file from the command line:
   ```
 
 - Once the build has finished, a file named `msbuild.binlog` is generated next to the `.csproj` file.
+
+## Common Build errors troubleshooting
+
+You can also browse the [common issues list](xref:Uno.UI.CommonIssues.AllIDEs) for build errors and their resolutions.
```

Comment:
And as a reverse thing, I think it would be best if the Common Issues per platform pages have a link to this troubleshooting page (xref: Uno.Development.Troubleshooting) as well no ?
![image](https://github.com/user-attachments/assets/7924465a-8da0-498f-8631-ea3972b42bed)

Reply:
There's no top level section for this, it's the overview, but we can add the link in all pages.